src/hyperlight_host/src/hypervisor/mod.rs

```diff
@@ -40,6 +39,9 @@ pub(crate) mod hyperv_windows;
 #[cfg(gdb)]
 pub(crate) mod gdb;
 
+/// Abstracts over different hypervisor register representations
+pub(crate) mod regs;
```

Comment:
Nit: I think it would be more appropriate to have an `arch` module that abstracts over registers and `architecture` specific logic.
This can be done later anyway.